packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDetailsTeleporterSteps.tsx

```diff
@@ -75,7 +75,7 @@ export const TransactionsTableDetailsTeleporterSteps = ({
 }) => {
   const { isTestnet: isTestnetTx } = isNetwork(tx.childChainId)
 
-  const l2TxID = tx.l1ToL2MsgData?.childTxId
+  const l2TxID = tx.parentToChildMsgData?.childTxId
```

Comment:
i don't think this should be changed at all

it is less accurate than what it was

unless we want to call l2 -> l3 childToGrandchild

Reply:
Since we have `l1ToL2MsgData` and `l2ToL3MsgData` in the types - I was reluctant at first too, but overall I agree this is a net positive change and teleporter is a special case. We need to be mindful of the fact that `parentToChildMsgData` only refers to L1-L2 in the teleporter context. Perhaps, it'd be nice to comment this wherever we're using this type?

`note: parentToChildMsgData refers to only to the l1-l2 leg in teleporter txns`